src/layer6.twee

```diff
@@ -761,6 +761,40 @@ You have successfully taken on the $curse77.name curse. Hopefully you are ok wit
 	[[Return to your balloon|Escape Balloon L6]]
 <</if>>
 
+:: Horny Customization
+What type of horn(s) would you like to gain?
+
+<<radiobutton "$curseData[77].variation" "demonic" checked>> Demon
+<<radiobutton "$curseData[77].variation" "draconic">> Dragon
+<<radiobutton "$curseData[77].variation" "unicorn">> Unicorn
+<<radiobutton "$curseData[77].variation" "rhino">> Rhino
+
+How many horns would you like to gain?
+
+<<radiobutton "$curseData[77].count" 1 checked>> One <<radiobutton "$curseData[77].count" 2 >> Two
+
+Enter one or more descriptive adjectives (optional)
```

Comment:
Maybe add some kind of example here, so players know how this will be used.

Reply:
Here's the examples I came up with so far:
![grafik](https://user-images.githubusercontent.com/13157984/177058728-c17274bd-2272-4156-851f-a397397c2dd7.png)

That should (at least for now) resolve the above, too.